packages/kit/types/index.d.ts

```diff
@@ -627,15 +635,17 @@ export interface ServerLoadEvent<
 
 export interface Action<
 	Params extends Partial<Record<string, string>> = Partial<Record<string, string>>,
-	OutputData extends Record<string, any> | void = Record<string, any> | void
+	OutputData extends Record<string, any> | void = Record<string, any> | void,
+	RouteId extends string | null = string | null
```

Comment:
do we need the overhead of that interface on the action interfaces? Actions can only be used on pages, so routeId can only be one thing, so it's useless to use it, so probably noone does that, so we probably can just type this as `string` on `RequestEvent`

Reply:
Is there any real drawback? It's contrived but you could imagine someone doing this sort of thing, where `some_shared_logic` is picky about the second argument:


```js
export const actions = {
  blah: ({ request, route }) => {
    some_shared_logic(request, route.id);
  }
};
```